data_validation/__main__.py

```diff
@@ -174,14 +174,18 @@ def get_calculated_config(args, config_manager: ConfigManager) -> List[dict]:
             if config_manager.hash == "*"
             else cli_tools.get_arg_list(config_manager.hash)
         )
-        fields = config_manager.build_dependent_aliases("hash", col_list)
+        fields = config_manager.build_dependent_aliases(
+            "hash", col_list, args.exclude_columns
```

Comment:
```suggestion
            "hash", col_list, config_manager.exclude_columns_list
```

Change will work only after having a new `CONFIG_EXCLUDE_COLUMNS_LIST` variable added to `consts.py` and a equivalent function added to `config_manager.py` to get it. Suggested structure for the function:


```python
    def exclude_columns_list(self):
        """Return if the validation should exclude list of columns provided instead of including"""
        return self._config.get(consts.CONFIG_EXCLUDE_COLUMNS_LIST) or False
```

Comment applies for lines 104, 111, 120, 129, 138, 147, 156, 187 as well

Reply:
Ah I forgot to mention: it seems this explicit call for args.exclude_columns can be the root cause behind the integration tests failure for MySQL

Reply:
The issue here was that the exclude_columns flag wasn't added to the arg parser - it is now! Similar to the existing column validation flag, it isn't stored in the config manager...it's only used to make the CLI easier. If you add the -ec flag, it won't be stored in the YAML config, but it will change the columns added to the YAML itself

Reply:
Ahhh got it! Makes more sense haha thanks Neha!